app/demo-geant-integration/PrimaryGeneratorAction.cc

```diff
@@ -11,28 +11,25 @@
 #include <G4SystemOfUnits.hh>
 #include <G4ThreeVector.hh>
 
+#include "corecel/io/Logger.hh"
+
+#include "HepMC3Reader.hh"
+
 namespace demo_geant
 {
 //---------------------------------------------------------------------------//
 /*!
- * Set up particle gun
+ * Construct empty.
  */
-PrimaryGeneratorAction::PrimaryGeneratorAction()
-{
-    auto g4particle_def = G4ParticleTable::GetParticleTable()->FindParticle(11);
-    gun_.SetParticleDefinition(g4particle_def);
-    gun_.SetParticleEnergy(500 * MeV);
-    gun_.SetParticlePosition(G4ThreeVector{0, 0, 0});          // origin
-    gun_.SetParticleMomentumDirection(G4ThreeVector{1, 0, 0}); // +x
-}
+PrimaryGeneratorAction::PrimaryGeneratorAction() {}
```

Comment:
Might as well leave this in the header with an `= default;` now that it's trivial.